custom_components/octopus_energy/sensor.py

```diff
@@ -583,6 +593,56 @@ def setup_heat_pump_sensors(hass: HomeAssistant, heat_pump_id: str, heat_pump_re
               sensor
             ))
 
+    if heat_pump_response.octoHeatPumpLivePerformance is not None:
+      entities.append(OctopusEnergyHeatPumpSensorLivePowerInput(
+        hass,
+        coordinator,
+        heat_pump_id,
+        heat_pump_response.octoHeatPumpControllerConfiguration.heatPump
+      ))
+
+      entities.append(OctopusEnergyHeatPumpSensorLiveHeatOutput(
+        hass,
+        coordinator,
+        heat_pump_id,
+        heat_pump_response.octoHeatPumpControllerConfiguration.heatPump
+      ))
+
+      entities.append(OctopusEnergyHeatPumpSensorLiveCoP(
+        hass,
+        coordinator,
+        heat_pump_id,
+        heat_pump_response.octoHeatPumpControllerConfiguration.heatPump
+      ))
+
+      entities.append(OctopusEnergyHeatPumpSensorLiveOutdoorTemperature(
+        hass,
+        coordinator,
+        heat_pump_id,
+        heat_pump_response.octoHeatPumpControllerConfiguration.heatPump
+      ))
+
+      entities.append(OctopusEnergyHeatPumpSensorLifetimeEnergyInput(
```

Comment:
suggestion: Based on the above, the lifetime based sensors should probably only be added if `lifetimeperformance` is not `none` instead of `octoHeatPumpLivePerformance`